src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2568,7 +2585,7 @@ private void decodeNetworkIfNecessary(
                 final int initialCredit = reserved - decodeSlotReserved;
                 if (initialCredit > 0)
                 {
-                    doNetworkWindow(traceId, authorization, initialCredit, 0, 0);
+                    doNetworkWindow(traceId, authorization, initialCredit, 0, 0, decodeSlotOffset, initialMax);
```

Comment:
We are passing `initialCredit` for `padding` here, likely it can be removed now that `maximum` parameter is gone and pass `0` for `padding` instead (as was the original intent, i think).

However, the logic of this still needs to be addressed given that `initialCredit` would no longer be used in the method call to `doNetworkWindow`.